cmd/kubectltranslate/kubectltranslate.go

```diff
@@ -52,6 +52,9 @@ func translateHandler(cmd *cobra.Command, flags cmdcommon.TranslateFlags) {
 	cmdcommon.CheckSourcePath(flags.Srcpath)
 	flags.Outpath = filepath.Join(flags.Outpath, flags.Name)
 	cmdcommon.CheckOutputPath(flags.Outpath, flags.Overwrite)
+	if flags.Srcpath == flags.Outpath || common.IsParent(flags.Outpath, flags.Srcpath) {
```

Comment:
Do we want to check the other way around too?

Reply:
`move2kube translate -s foo/bar/ -o foo/` should be fine

Reply:
Exactly, we should be checking the final output folder, along with project name